overrides/android-override.json

```diff
@@ -195,6 +195,16 @@
                 },
                 "screenLock": {
                     "state": "enabled"
+                },
+                "optimizeTrackerEvaluation": {
```

Comment:
@CrisBarreiro please add a base file for this feature in the `features` directory. I'm out of office right now but you can find more information in the "Privacy Remote Configuration" project in Asana.

Reply:
Thanks @SlayterDev! This is a subfeature, not a feature. Base file for the feature already exists in https://github.com/duckduckgo/privacy-configuration/blob/main/features/androidBrowserConfig.json